src/ServicePulse.Host/app/js/services/services.monitoring.js

```diff
@@ -2,27 +2,36 @@
 (function (window, angular, $, undefined) {
     'use strict';
 
-    function Service($http, scConfig, notifications, uri) {
-
-        function getRaw() {
-            var url = uri.join(scConfig.monitoring_url, '/raw');
-            return $http.get(url)
-                .then(function (response) {
-                    return response.data;
-                })
-                .catch(function(fallback) {
-                    notifications.pushSticky('Can not connect to Monitoring Service');
+    function Service($http, rx, scConfig, uri) {
+
+        function getEndpoints() {
+            return rx.Observable.merge(scConfig.monitoring_urls.map(function (url) {
+                var requestUri = uri.join(url, '/raw');
+
+                var httpRequest = rx.Observable.just(url)
+                    .flatMap(function (requestUrl) {
+                        return $http.get(requestUrl);
+                    });
+
+                var repeatRequests = rx.Observable.interval(5000).flatMapLatest(httpRequest.retry().delay(5000));
```

Comment:
The `retry()` just means if it fails, try again in another 5 seconds. It's actually a real pain to get the continual retry stuff working properly. I suspect it's not working correctly at the moment anyway.

Scenarios we need to support:

* Service is configured but is restarting or temporarily unavailable
    * This should auto-retry until connected again
* Service is configured but not available
    * Should probably show a notification to the user?
    * Same for incorrectly configured service

So thinking about it while typing this, there's only 1 viable use case for the retries, and those should probably be limited to, say, 10 retries max before stopping?